products/openbsd.md

```diff
@@ -1,237 +1,247 @@
 ---
+# Product level information
+
 title: OpenBSD
 category: os
 tags: bsd-distribution
 iconSlug: openbsd
 permalink: /openbsd
 versionCommand: uname -r
 changelogTemplate: "https://www.openbsd.org/{{'__RELEASE_CYCLE__'|replace:'.',''}}.html"
-activeSupportColumn: false
+
+# Formatting
+
 releaseDateColumn: true
+eolColumn: true
+activeSupportColumn: false
 releaseColumn: false
 
+# eol(x) = releaseDate(x+2), Estimation releaseDate(x) + 1 year -> round to first of next month
+
+# Release Cycles
+
 releases:
--   eol: 2024-11-01
+-   eol: 2024-11-01  # estimated from releaseDate: 2023-10-16
     releaseCycle: "7.4"
     releaseDate: 2023-10-16
```

Comment:
Please check the order in [Style-Guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide), should be `releaseCycle`, `releaseDate`, `eol`
```suggestion
-   releaseCycle: "7.4"
    releaseDate: 2023-10-16
    eol: 2024-11-01  # estimated from releaseDate: 2023-10-16
```
Please update all below cycles.